OpenSim/ExampleComponents/Device.h

```diff
@@ -0,0 +1,94 @@
+#ifndef _OPENSIM_DEVICE_H_
+#define _OPENSIM_DEVICE_H_
+/* -------------------------------------------------------------------------- *
+ *               OpenSim:  defineDeviceAndController_answers.h                *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2017 Stanford University and the Authors                *
+ * Author(s): Chris Dembia, Shrinidhi K. Lakshmikanth, Ajay Seth,             *
+ *            Thomas Uchida                                                   *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include <OpenSim/OpenSim.h>
+
+namespace OpenSim {
+
+//------------------------------------------------------------------------------
+// Device is a type of ModelComponent that contains all the parts comprising the
+// assistive device model (a PathActuator plus bodies and joints).
```

Comment:
> I have copied the full description from the original code.

I assumed that you had. My point is that there isn't enough information here for someone to use this Component elsewhere without knowledge of the hopper example. I don't know what the use case is for this new library, but it seems like there are two options here:
(a) Make this component a generic device, and provide enough information for someone to use it without assuming they're familiar with other examples.
(b) Make this component specific to the hopper example, and provide a comment at the top indicating this and telling the user where the hopper example is.